DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/RemoveRegistrationPrompt.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CentreConfiguration
+@using Microsoft.Extensions.Configuration
+@model RemoveRegistrationPromptViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Remove delegate registration prompt" : "Remove delegate registration prompt";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+  var confirmError = ViewData.ModelState[nameof(RemoveRegistrationPromptViewModel.Confirm)]?.Errors?.Count > 0;
+  var confirmFormErrorClass = confirmError ? "nhsuk-form-group--error" : "";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@if (errorHasOccurred) {
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+      <vc:error-summary order-of-property-names="@new []{ nameof(RemoveRegistrationPromptViewModel.Confirm) }" />
+    </div>
+  </div>
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Remove delegate registration prompt</h1>
+  </div>
+</div>
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l">
+    <div class="nhsuk-u-font-weight-bold">
+      Prompt:
+    </div>
+  </div>
+  <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l nhsuk-u-font-weight-normal">
+    <input type="hidden" asp-for="PromptName" />
+    @Model.PromptName
+  </div>
+</div>
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full nhsuk-body-m">
+    Some users have answered this prompt. By deleting this prompt, all the answers that the delegates have given will be deleted too.

```

Comment:
Should this be in a <p> element?

Reply:
Not sure what you mean here, you need to escape the \<p> or whatever you've put with a \